brion/morphologyPlugin.h

```diff
@@ -144,6 +141,9 @@ class MorphologyPlugin : public boost::noncopyable
     /** @copydoc brion::Morphology::flush */
     virtual void flush() = 0;
     //@}
+
+protected:
+    MorphologyInitData _data;
```

Comment:
not sure about this, it seems preferable to keep a pure abstract base class without data members